app/views/admins/_user_entry.mobile.haml

```diff
@@ -0,0 +1,70 @@
+
+%li.user.media
+  %div.pull-left
+    - if user.person
+      %span.media-object
+        = person_image_tag(user.person, size: :thumb_small)
+
+  %div.media-body.row
+    %div.pull-right
+      %span.label.label-default
+        = t('.id')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.